cocos/asset/assets/texture-2d.ts

```diff
@@ -94,6 +96,36 @@ export class Texture2D extends SimpleTexture {
         return this._mipmaps;
     }
     set mipmaps (value) {
+        if (value[0].mipmapLevelDataSize && value[0].mipmapLevelDataSize.length > 0) {
+            compressedImageAsset.length = 0;
+            const mipmapLevelDataSize = value[0].mipmapLevelDataSize;
+            const data: Uint8Array = value[0].data as Uint8Array;
+
+            let byteOffset = 0;
+            for (let i = 0; i < mipmapLevelDataSize.length; i++) {
+                const srcView = new Uint8Array(data.buffer, byteOffset, mipmapLevelDataSize[i]);
+                const dstView = new Uint8Array(mipmapLevelDataSize[i]);
+                dstView.set(srcView);
```

Comment:
Why need to copy to a new dstView? Could we just use srcView to avoid new memory being allocated?

Reply:
We have tried to get a piece of the srcView as data to upload to the GPU, but the data is unexpectedly lost when uploading GPU resources this way. We suspect that the dataView only references this piece of data and is not a deep copy, which is why the reference release data is lost. This is how we used it in the beginning, and the error bothered us for a long time, so we had to do a deep copy to solve the problem